x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -543,13 +570,35 @@ export class ExecutionHandler<
     const alerts = await this.ruleType.getSummarizedAlerts!(options);
 
     const total = alerts.new.count + alerts.ongoing.count + alerts.recovered.count;
-    return {
+    const summarizedAlerts = {
       ...alerts,
       all: {
         count: total,
         data: [...alerts.new.data, ...alerts.ongoing.data, ...alerts.recovered.data],
       },
     };
+
+    if (summarizedAlerts.all.count > 0) {
```

Comment:
Returns the time bounds for summarized alerts that can be used for building rule URLs. This ensures that the time bounds used to load alerts in the UI matches the time bounds for the alert summary so the alert counts match.